lib/bulk/unordered.js

```diff
@@ -47,7 +47,7 @@ var FindOperatorsUnordered = function(self) {
  * @method
  * @param {object} doc update operations
  * @throws {MongoError}
- * @return {UnorderedBulkOperation}
+ * @return {FindOperatorsUnordered}
  */
 FindOperatorsUnordered.prototype.update = function(updateDocument) {
```

Comment:
it's not a hard requirement, but I think on a "as I'm working in the area" rule we can make sure the jsdoc parameter names match up with the actual name used in the function. `doc` => `updateDocument` in this case, what do you think?